apps/coscrad-frontend/src/components/dynamic-forms/dynamic-form-elements/year-picker.tsx

```diff
@@ -0,0 +1,43 @@
+import { IFormField, IValueAndDisplay } from '@coscrad/api-interfaces';
+import { StaticSelect } from './static-select';
+
+// TODO Consider making this configurable
+const STARTING_DATE_TO_PICK = 1491;
+
+// TODO Share this with the validator lib
+const CURRENT_YEAR = new Date().getFullYear();
+
+interface YearPickerProps {
+    formField: IFormField;
+}
+
+/**
+ * Note that MUI has a free and paid teir for their date-picker element, which
+ * is not part of their core library. We decided it was safest to simply
+ * "role our own".
+ */
+export const YearPicker = ({
```

Comment:
TODO component test

Reply:
[Here](https://www.pivotaltracker.com/story/show/184066046) is a story for that.